### DISCUSSION TITLE
I think posted in wrong place but cannot delete it

### DISCUSSION THREAD

**Discussion by jofori** [atom/atom#22139] [Q&A]

**Comment by ThatXliner** [comment]
Whups- I mean, you can't close or delete a Github Discussion yet. But you *can* answer questions, like this one. 


Just click on the accept answer button

**Reply by jofori** [reply]
which one? I don't see anything

**Reply by ThatXliner** [reply]
That button
<img width="950" alt="Screen Shot 2021-04-06 at 9 47 28 AM" src="https://user-images.githubusercontent.com/66848002/113748301-2a718480-96bd-11eb-96a3-3c209819a6d9.png">
